tracer/test/snapshots/WcfTests.__metadataSchemaVersion=v0_binding=WSHttpBinding_enableNewWcfInstrumentation=True_enableWcfObfuscation=True.verified.txt

```diff
@@ -31,7 +31,7 @@
     TraceId: Id_1,
     SpanId: Id_4,
     Name: wcf.request,
-    Resource: WcfSample/ICalculator/ServerSyncAdd,
+    Resource: WcfSample/ICalculator/ServerAsyncAdd,
```

Comment:
I was trying to understand the trace and realised that this only includes _half_ of the spans so [my snapshot viewer doesn't work](https://andrewlock.github.io/DatadogSpanSnapshotViewer/) 😢  Taking your word for it that these are correct in that case 😉